pymdb.py

```diff
@@ -129,17 +121,17 @@ def year(self):
     def rating(self):
         """Prints IMDb and RT ratings"""
         return float(self.stuff.find('movie').get("imdbRating"))
-        
+
     def RTrating(self):

```

Comment:
Function name doesn't conform to PEP8 standards (`snake_case`).

`RTrating -> rt_rating`